lib/Clients/Service/Localise.php

```diff
@@ -37,20 +36,31 @@ public function authenticate( array $args ) {
 		// Clean previous client.
 		$this->client = null;
 
-		// Make a new client.
-		$client = ApiClient::factory( $args );
+		$client = new \GuzzleHttp\Client(
+			[
+				'headers' => [
+					'Authorization' => 'Loco ' . $args['key'],
+				],
+			]
+		);
 
-		// Try to authenticate it.
-		try {
-			$result = $client->authVerify();
-			printf( "Authenticated as '%s'.\n", $result['user']['name'] );
-		} catch ( Exception $e ) {
+		$res = $client->request(
+			'GET',
+			'https://localise.biz/api/auth/verify',
```

Comment:
Deves passar o `https://localise.biz/api/` como base_uri na instanciação do `Client`.

```suggestion
			'auth/verify',
```